tests/unit/utils/add-pagination-meta-test.js

```diff
@@ -0,0 +1,32 @@
+import addPaginationMeta from '@bagaaravel/ember-data-extensions/utils/add-pagination-meta'
+import { module, test } from 'qunit'
+
+module('Unit | Utility | add-pagination-meta', function () {
+  test('it works', function (assert) {
```

Comment:
Maybe it should also be tested that it doesn't do anything if not all links are valid?